### ISSUE TITLE
AdminThemeUikit - Fix AOS Settings Page module boxes

### ISSUE THREAD

**Issue by gmclelland** [rolandtoth/AdminOnSteroids#71]
When you visit processwire/module/edit?name=AdminOnSteroids&collapse_info=1 this is what it looks like:

![modules processwire pwtracy test 2017-10-19 17-19-25](https://user-images.githubusercontent.com/40570/31797035-c43ce20a-b4f1-11e7-9655-8f9a2815a32d.jpg)

Fix is coming soon

**Comment by rolandtoth** [comment]
I have fixed this in another way but will check yours too (unreleased yet).

**Comment by gmclelland** [comment]
Just documenting for future reference if needed...I forgot to mention with my solution you also need to add:
outline:none to:

line 187 of aos_config.scss
```css
li.InputfieldAsmSelect {
    position: static;
    margin: 0 !important;
    background: transparent !important;
    outline: none; 
}
```
That gets rid of that weird underline:
![modules](https://user-images.githubusercontent.com/40570/31970515-d1e2eb76-b8dd-11e7-81af-19c4dfbb572e.jpg)

So now it looks like:
![modules processwire pwoliver test 2017-10-24 17-08-23](https://user-images.githubusercontent.com/40570/31970554-f6b72c46-b8dd-11e7-9807-1c99ee59751b.jpg)

**Comment by rolandtoth** [comment]
Thanks, I have added these already. The cog icons are positioned too lower because some margins, that was also fixed. I'll try to put the updates online tomorrow so you can see what have I missed to merge :)

**Comment by rolandtoth** [comment]
Fixed in v1.6.3, thanks!
